File: week-02/day-02/printer.py
# - Create a function called `printer`
#   which prints the input parameters
#   (can have multiple number of arguments)
def printer(*args):
    oneline = ""
    for i in args:
        oneline += str(i)
        oneline += " "
        print(i)
    print(oneline)
printer("Hello", "Attila", 7)

File: week-07/day-02/printer.js
'use strict';
// - Create a function called `printer`
//   which logs to the console the input parameters
//   (can have multiple number of arguments)

function printer() {
  for (let i=0; i < arguments.length; i++) {
    console.log(arguments[i]);
  }
}

printer("hello", "bello");